packages/amazon-sumerian-hosts-babylon/examples/hello-world-demo.js

```diff
@@ -0,0 +1,49 @@
+import DemoUtils from './common/demo-utils.js';
+import cognitoIdentityPoolId from './common/demo-credentials.js';
+
+let host;
+let scene;
+
+async function createScene() {
+  // Create an empty scene. IMPORTANT: Sumerian Hosts require use of the
+  // right-hand coordinate system!
+  scene = new BABYLON.Scene();
+  scene.useRightHandedSystem = true; // IMPORTANT for Sumerian Hosts!
```

Comment:
You mentioned this before. Excuse my ignorance: what does this mean for customers with existing scenes? are they able to add a host to it?

Reply:
If we're doing this for gltf loading, it looks like the GLTFLoader is going to automatically hand this for us with flag GLTFLoaderCoordinateSystemMode default as AUTO

Reply:
@prestomation My understanding is that although _some_ existing scenes might be able to be adapted easily (static scenes in particular), many others won't. My hunch that a small change to our code could remove this right-hand coordinate system requirement, but I'm not sure. And I've considered it outside the scope of this PR.

Sounds like maybe Runping has some insight into a future potential fix?

Reply:
BTW, the real-world impact of _not_ setting the coordinate system to right-handed is that the host's point-of-interest tracking works backwards. They look left when they should be looking right to track a target.

Reply:
It seems to me we should swap the default in our hosts, so default babylon setup 'just works', but then make this configurable. otherwise we will have the opposite problem, customers using a right handed system(due to some other dependency they have) will then have a problem. we should default to whatever is more popular, which I assume is babylon's default.

Reply:
I'm very much in agreement. I've created a new internal ticket for us to track this desired improvement. SUMERIAN-11086